docs/sources/github/copilot/github-copilot.yaml

```diff
@@ -480,9 +480,7 @@ endpoints:
       - !<tokenize>
         jsonPaths:
           - "$..slug"
-  - pathRegex: "^/orgs/[^/]+/audit-log\\?(?=[A-Za-z0-9=&%+:\\._\\-]*phrase=action:copilot(?:\\\
-    +created=[^&]+)?)(?=[A-Za-z0-9=&%+:\\._\\-]*include=web)[A-Za-z0-9=&%+:\\._\\\
-    -]+$"
+  - pathRegex: "^/orgs/[^/]+/audit-log\\\\?(?=.*[\\\\?&]phrase=)(?=.*[\\\\?&]include=).*$"
```

Comment:
i don't think you can have BOTH a `pathRegex` and a `pathTemplate` ... or at least, that shouldn't work.